coalib/processes/BearRunning.py

```diff
@@ -408,13 +397,27 @@ def run_local_bears(filename_queue,
     try:
         while True:
             filename = filename_queue.get(timeout=timeout)
+
+            if filename not in file_dict:
+                send_msg(message_queue,
+                         timeout,
+                         LOG_LEVEL.ERROR,
+                         "An internal error occurred.",
+                         Constants.THIS_IS_A_BUG)
+                send_msg(message_queue,
+                         timeout,
+                         LOG_LEVEL.DEBUG,
+                         "The given file through the queue is not in the file "
+                         "dictionary.")
+                continue
```

Comment:
actually this shouldn't happen at all, can you remove that branch altogether? If the filename isn't in there the exception is the right thing to get thrown.